search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt

```diff
@@ -194,7 +194,7 @@ class V0_28__JsonLd_migration : BaseJavaMigration() {
                             ${temporalAttributesMetadata.value.measuredValue}, 
                             ${temporalAttributesMetadata.value.value.toSQLValue()}, 
                             ST_GeomFromText('${temporalAttributesMetadata.value.geoValue!!.value}'), 
-                            '$teaId', '$attributeInstanceId', '$serializedAttributePayload')
+                            '$teaId', $$$attributeInstanceId$$, '$serializedAttributePayload')
```

Comment:
`attributeInstanceId` is generated by us and does not need to be dollar-quoted. But `serializedAttributePayload` should.